Tests/ClientTests/Wallpaper/UtilitiesTests/DataService/WallpaperDataServiceMock.swift

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+@testable import Client
+
+class WallpaperDataServiceMock: WallpaperFetchDataService {
+
+    private var mockNetworkResponse: Result<WallpaperMetadata, Error>?
```

Comment:
Personally I'd remove the private and the setServiceResponse method and allow the response to be set directly to the var considering this is a mock.